app/services/gtfs_graph.rb

```diff
@@ -127,54 +134,56 @@ def log(msg)
 
   def load_tl_stops
     # Merge child stations into parents.
-    log "  merge stations"
-    stations = Hash.new { |h,k| h[k] = [] }
+    log '  merge stations'
+    stations = Hash.new { |h, k| h[k] = [] }
     @gtfs.stops.each do |stop|
       stations[@gtfs.stop(stop.parent_station) || stop] << stop
     end
     # Merge station/platforms with Stops.
-    log "  stops"
-    stations.each do |station,platforms|
+    log '  stops'
+    stations.each do |station, platforms|
       # Temp stop to get geometry and name.
       stop = Stop.from_gtfs(station)
       # Search by similarity
-      stop, score = Stop.find_by_similarity(stop[:geometry], stop.name, radius=1000, threshold=0.6)
+      stop, = Stop.find_by_similarity(
+        stop[:geometry],
+        stop.name,
+        radius: 1000,
+        threshold: 0.6
+      )
       # ... or create stop from GTFS
       stop ||= Stop.from_gtfs(station)
       # ... check if Stop exists, or another local Stop, or new.
       stop = find_by_entity(stop)
       # Add identifiers and references
-      ([station]+platforms).each { |e| add_identifier(stop, 's', e) }
+      ([station] + platforms).each { |e| add_identifier(stop, 's', e) }
       # Cache stop
-      if score
-        log "    #{stop.onestop_id}: #{stop.name} (search: #{station.stop_name} = #{'%0.2f'%score.to_f})"
-      else
-        log "    #{stop.onestop_id}: #{stop.name}"
-      end
+      log "    #{stop.onestop_id}: #{stop.name}"
     end
   end
 
   def load_tl_routes
     # Routes
-    log "  routes"
+    log '  routes'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.